investor_wallet_platform_base/models/coop.py

```diff
@@ -72,6 +72,17 @@ def default_structure(self):
                                 domain=[('is_platform_structure', '=', True)],
                                 default=default_structure)
 
+    @api.model
+    def total_outstanding_amount(self):
+        """Compute the total outstanding amount for subscriptions."""
+        subscriptions = self.search([("type", "=", "subscription")])
+        sell_backs = self.search([("type", "=", "sell_back")])
+        amount = (
+            sum(r.total_amount_line for r in subscriptions)
+            - sum(r.total_amount_line for r in sell_backs)
+        )
+        return amount
+

```

Comment:
@remytms here is was it's said on the task for this requirements 

"Le décompte de l’encours (= volume que représente l’investissement total dans les structures affichées sur le site vitrine, mis à jour deux fois par an manuellement)."

My understanding is that it should come from a field filled by the structure manager on the structure itself.

The two other counters will be fields res_company that are computed.

Reply:
J'avais compris qu'il devait être calculé et qu'il serait mis à jour 2 fois par an sur le site vitrine à la main.

Donc j'ajoute un champ sur la structure pour que le gérant de la structure puisse changé la valeur 2 fois par ans.

Je déplace les deux autres calcul dans des champs calculé sur le res_company.

Reply:
Mais il faut quand même que le total de l'encours soit calculé par odoo si j'ai bien compris. Donc ce calcul de l'encours doit être placé sur le `res_company` et se baser sur les valeurs entrées manuellement sur chaque structure.